src/resolve/transform.ts

```diff
@@ -0,0 +1,25 @@
+import { TwTheme } from '../tw-config';
+import { StyleIR, TransformType } from '../types';
+import { transformStyle, parseStyleVal, parseUnconfigged } from '../helpers';
+
+type InsetDir = null | 'x' | 'y';
+
+export function transform(
+  type: TransformType,
+  value: string,
+  isNegative: boolean,
+  // config?: TwTheme['inset'],
+): StyleIR | null {
+
+  // const configValue = config?.[value];
+  // if (configValue) {
+  //   const styleVal = parseStyleVal(configValue, { isNegative });
+  //   if (styleVal !== null) {
+  //     return insetStyle(type, insetDir, styleVal);
+  //   }
+  // }
+
+  return transformStyle({
+    rotate: `${isNegative ? '-' : ''}${value}deg`
```

Comment:
I'm a little confused as to why an new unit "absolute" was introduced, when it seems that the unit is actually `deg`.

Reply:
that's a fair point. it's been added because the tailwind api for transform is `rotate-45`, `scale-95` without any unit, so this needs to be parsed i think?